bin/ColorScript/Sources/ColorScriptCore/Sources/ColorScriptCore.swift

```diff
@@ -0,0 +1,118 @@
+// swiftlint:disable force_unwrapping
+import Foundation
+
+extension Dictionary {
+  public func withAllValuesFrom(_ other: Dictionary) -> Dictionary {
+    var result = self
+    other.forEach { result[$0] = $1 }
+    return result
+  }
+}
+
+public struct Color {
+
+  public let data: Data
+
+  public init(data: Data) {
+    self.data = data
+  }
+
+  public func colors() -> [(key: String, value: String)]? {
+    do {
+      if let json = try (JSONSerialization.jsonObject(with: data, options: []) as? [String: String]) {
+        let colors = json
+          .map { (key: $0, value: $1) }
+          .sorted { $0.key < $1.key }
+        return colors
+      }
+      return nil
+    } catch {
+      return nil
+    }
+  }
+
+  public var prettyColors: String {
+    return colors()!.map { (color, value) in
```

Comment:
Might want to handle the optionality of `colors()` here and also refer to `self`.